compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -797,7 +785,7 @@ class Namer { typer: Typer =>
         denot.info = UnspecifiedErrorType
       }
       else {
-        completeInContext(denot, ctx)
+        completeInContext(denot, this.ctx)
```

Comment:
Could you please add a comment saying that our "default behaviour" is to complete using the creation context?